lib/hawk/model/association.rb

```diff
@@ -18,47 +18,88 @@ def initialize(attributes = {}, params = {})
         super
         if attributes.size > 0 && self.class.associations?
           preload_associations(attributes, params, self.class)
+
+          # Handle assocs with no entries
+          params[:links].to_s.split(/,/).each do |name|
```

Comment:
Also the splitting by ',' is a Reticulum detail. We should expect params[:includes] is an Array.